packages/cms-data-sync/src/teams/teams.data-migration.ts

```diff
@@ -0,0 +1,76 @@
+import { SysLink } from 'contentful-management';
+import { teamsQuery } from './teams.queries';
+import {
+  FetchTeamsQuery,
+  FetchTeamsQueryVariables,
+} from '../autogenerated-gql/graphql';
+import {
+  clearContentfulEntries,
+  createExternalToolLinks,
+  getSquidexAndContentfulClients,
+  logger,
+} from '../utils';
+import { migrateFromSquidexToContentfulFactory } from '../utils/migration';
+
+type TeamItem = NonNullable<FetchTeamsQuery['queryTeamsContents']>[number];
+
+export const migrateTeams = async () => {
+  const { contentfulEnvironment, squidexGraphqlClient } =
+    await getSquidexAndContentfulClients();
+
+  await clearContentfulEntries(contentfulEnvironment, 'externalTools');
+
+  const migrateFromSquidexToContentful = migrateFromSquidexToContentfulFactory(
+    contentfulEnvironment,
+    logger,
+  );
+
+  const fetchData = async () => {
+    const gqlTeams = await squidexGraphqlClient.request<
+      FetchTeamsQuery,
+      FetchTeamsQueryVariables
+    >(teamsQuery);
+    return gqlTeams.queryTeamsContents || [];
+  };
+
+  const parseTeamItem = async (team: TeamItem) => {
+    const { flatData: squidexTeamItem, id } = team;
+
+    const {
+      applicationNumber,
+      displayName,
+      expertiseAndResourceTags,
+      inactiveSince,
+      projectSummary,
+      projectTitle,
+      tools,
+    } = squidexTeamItem;
+
+    let toolLinks: SysLink[] = [];
+    if (tools?.length) {
+      try {
+        toolLinks = await createExternalToolLinks(contentfulEnvironment, tools);
+      } catch {
+        logger(`Invalid tool URL linked to team with id: ${id}`, 'ERROR');
```

Comment:
there is still one issue with this :D it will reject all links if only one fails, no? i think we wanna be able to insert the valid ones